doc/Makefile

```diff
@@ -67,11 +67,7 @@ html:
 	@echo
 	@echo "Build finished. The HTML pages are in $(BUILDDIR)/html."
 
-develop:
-	FAST=True $(SPHINXBUILD) -b html $(ALLSPHINXOPTS) $(BUILDDIR)/html
-	@echo
-	@echo "Build finished. The HTML pages are in $(BUILDDIR)/html."
```

Comment:
is there a reason we need to remove these messages? It was actually one of the first PRs I made because devs can never remember the path to see the rendering of the local build.

Reply:
Ah, I see! Actually, the line I added was line 74. If that still applies, can we add that to `html`?

Reply:
Sure!